docs/source/model_design/vivarium_model_components/fertility/index.rst

```diff
@@ -0,0 +1,134 @@
+.. _fertility_live_births:
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  ---------------------
+
+  Section Level 2 (#.#)
+  +++++++++++++++++++++
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+==============================================
+Fertility Component that uses Live Births data
+==============================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Overview
+-------------
+
+This document is intended to represent the fertility model for a simulation that adds new simulants based on live births data,
+rather than by identifying which simulants give birth.
+
+GBD Modeling Strategy
+----------------------
+
+Pregnancy and births are not explicit outcomes in the GBD study. However, there are location- and year-specfic GBD covariates related to fertility and births which we can use to calculate the number
+of new simulants to add per timestep.
+
+.. list-table:: Covariates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Covariate name
+    - Covariate ID
+    - Note
+  * - Live births by sex
+    - 1106
+    - 
+
+
+Vivarium Modeling Strategy
+----------------------------
+
+In a simulation where we do not model births by identifying which simulants give birth, we must instead use live births data
+to determine how many simulants to add in each time step.
+We will scale down the live births of the total GBD population by the same "model scale"  we use in our simulation.
+
+For example, imagine that country A has a GBD population of 1000, and we are modeling this total population, but only need to initialize children age 5 and under 
+(e.g. because they are the only ones susceptible to the disease we are modeling).
+Imagine there are 100 such children living in A, and we plan to initialize 20 children in our model. 
+
+In this example, our initialized population is :math:`\frac{1}{5}` the size of the age-restricted GBD population it corresponds to. 
```

Comment:
I think it would be good to explicitly explain where you got 1/5:
```suggestion
In this example, our initialized population is :math:`\frac{1}{5}` the size of the age-restricted GBD population it corresponds to, i.e., 20 simulated children under 5 vs. 100 actual children under 5.
```
(Or switch to the bigger numbers I suggested above.)